miden-lib/src/transaction/mod.rs

```diff
@@ -20,7 +20,7 @@ use super::MidenLib;
 pub mod memory;
 
 mod events;
-pub use events::{TransactionEvent, TransactionTrace};
+pub use events::{TransactionEvent, TransactionTrace, EVENT_ID_PREFIX};
```

Comment:
Would it make sense to associate `EVENT_ID_PREFIX` with `TransactionEvent`? This way, we would be able to say something like `TransactionEvent::ID_PREFIX` and wouldn't need to re-export the constant separately from here.